lib/codeowners/cli/main.rb

```diff
@@ -143,10 +142,10 @@ def suggest_subgroups_for_pattern(subgroups)
       end
 
       def suggest_fix_for(line)
-        search = FuzzyMatch.new(line.suggest_files_for_pattern)
-        suggestion = search.find(line.pattern)
+        return unless options[:interactive]
 
-        puts "Pattern #{line.pattern.inspect} doesn't match." if options[:interactive]
+        puts "Pattern #{line.pattern.inspect} doesn't match."
```

Comment:
Not sure if we should always show this message if it's not interactive. I guess printing always is not a good idea since the checker iterate under results.